WordPress/src/main/java/org/wordpress/android/ui/notifications/blocks/CommentUserNoteBlock.kt

```diff
@@ -23,29 +21,29 @@ import org.wordpress.android.util.image.ImageType
 // A user block with slightly different formatting for display in a comment detail
 @Suppress("LongParameterList")
 class CommentUserNoteBlock(
```

Comment:
Most of the changes are related to renaming variables and removing some unused functions in this file.